models/src/main/scala/coop/rchain/models/rholang/implicits.scala

```diff
@@ -494,4 +494,42 @@ object implicits {
           case _ => BitSet()
         }
     }
+
+  implicit val ParIsForgeable: IsForgeable[Par] =
```

Comment:
I think those are unintended changes, and the CI build seems to agreee :)

Reply:
I've fixed it so those aren't there.